src/test/dbwrapper_tests.cpp

```diff
@@ -73,7 +74,7 @@ BOOST_AUTO_TEST_CASE(dbwrapper_batch)
         BOOST_CHECK(dbw.Read(key2, res));
         BOOST_CHECK_EQUAL(res.ToString(), in2.ToString());
 
-        // key3 never should've been written
+        // key3 should've never been written
```

Comment:
I could nitpick and say that this should be "key3 should never have been written", but I'll spare you that :-)